script/release/publish-to-npm.js

```diff
@@ -161,6 +162,26 @@ new Promise((resolve, reject) => {
       }
     }
   })
+  .then(() => {
+    const DEPSContent = fs.readFileSync(path.resolve(__dirname, '..', '..', 'DEPS'), 'utf8');
+
+    const getVersion = (dep) => {
+      // e.g: /'chromium_version':(\W+)(.+?)'/
+      const re = new RegExp(`'${dep}':(\\W+)(.+?)'`);
+      return re.exec(DEPSContent)[2];
+    };
+
+    const chromiumVersion = getVersion('chromium_version');
+    // v16.0.0 => 16.0.0
+    const nodeVersion = getVersion('node_version').slice(1);
+
+    fs.writeFileSync(path.join(tempDir, 'versions.json'), JSON.stringify({
+      chromium: chromiumVersion,
+      node: nodeVersion,
+      chromiumShort: chromiumVersion.split('.')[0],
+      nodeShort: nodeVersion.split('.')[0]
```

Comment:
It's just done here in passing.

Reply:
I think it's best to leave this to user land because exposing the major part of the version separately gives a good enough reason to expose the minor part, the patch part and so on, which would lead to unnecessary bloating of the json, which is not ideal

Reply:
okey